src/Microsoft.ApiDesignGuidelines.Analyzers/CSharp/CSharpEnumStorageShouldBeInt32.Fixer.cs

```diff
@@ -3,14 +3,21 @@
 using System.Composition;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
 
 namespace Microsoft.ApiDesignGuidelines.Analyzers
 {
-    /// <summary>
-    /// CA1028: Enum Storage should be Int32
-    /// </summary>
+    /// <summary> 
+    /// CA1028: Enum Storage should be Int32 
+    /// </summary> 
```

Comment:
Not that I care, but I'm curious why the VS editor would have added this trailing white space.